packages/cw0/src/pagination.rs

```diff
@@ -11,32 +11,20 @@ pub fn maybe_addr(api: &dyn Api, human: Option<String>) -> StdResult<Option<Addr
 }
 
 // this will set the first key after the provided key, by appending a 0 byte
-pub fn calc_range_start_human(
-    api: &dyn Api,
-    start_after: Option<Addr>,
-) -> StdResult<Option<Vec<u8>>> {
-    match start_after {
-        Some(human) => {
-            let mut v: Vec<u8> = api.addr_canonicalize(human.as_ref())?.0.into();
-            v.push(0);
-            Ok(Some(v))
-        }
-        None => Ok(None),
-    }
+pub fn calc_range_start(start_after: Option<Addr>) -> Option<Vec<u8>> {
```

Comment:
Can only do proper / full reviews beginning next week.

Perhaps the original functions must be kept, as they can be useful for old-style ranges / storage. These new ones can also be suffixed by `_addr` for clarity, but given that this is the new standard, I don't think that's really necessary.

Reply:
I see now that the original argument is `Addr` already so, it's OK to remove / rename these functions.